_episodes/04-data-types-and-format.md

```diff
@@ -15,7 +15,11 @@ objectives:
     - "Analyze datasets having missing/null values (NaN values)."
     - "Write manipulated data to a file."
 keypoints:
-    - "FIXME"
+    - "Pandas and base Python use slightly different names for data types"
+    - "Each column in a DataFrame has a shared data type for all rows"
```

Comment:
... Is there an easier way to convey the same information? ... I mean, I understand what you're trying to say here because I know what you mean. But "shared data type" does not sound right. I suggest we add two points instead: about rows and about columns. About columns you can say something like `All data in a column in a DataFrame are of the same type` or `A column in a DataFrame can have data of one type only`

Reply:
I like "A column in a DataFrame can only have one data type".